components/backup/production/kflux-prd-rh02/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/all-clusters
+  - ../../base/host/schedules
+  - ../../base/member/schedules
```

Comment:
as mentioned above, we are not using toolchain host/member on this cluster so we need to include member schedules but we only need one of the two from that folder, we only need the tenant one